core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -83,7 +83,7 @@ SYCL::SYCL() : m_space_instance(&Impl::SYCLInternal::singleton()) {
 
 int SYCL::concurrency() {
   // FIXME_SYCL
-  return 1;
+  return 2;
```

Comment:
This is still a fixme but one of the test was failing because asserting that the concurrency is greater than one

Reply:
I'd like to see a more detailed comment than "FIXME" here, something like "this is to work around some tests that assert concurrency must be greater than 1"